src/lib/content.ts

```diff
@@ -2,9 +2,19 @@
 import type { PRDetails } from './github';
 import * as core from '@actions/core';
 
-function removeDeletedFilter(file: File) {
+// function removeDeletedFilter(file: File) {
+//   const conditions = [
+//     file.to === '/dev/null',
+//     file?.chunks?.length <= 0
+//     //...
+//   ];
+
+//   return !conditions.some(condition => condition);
+// }
+
+function removeInvalidFilter(file: File) {
   const conditions = [
-    file.to === '/dev/null',
+    // file.to === '/dev/null',
```

Comment:
📌
A linha comentada 'file.to === '/dev/null'' foi substituída por uma lógica no novo código que verifica se 'file.to' é igual à '/dev/null'. A condição comentada é redundante e pode ser removida para evitar confusão.
Remover código comentado que foi substituído ajuda a evitar confusões e torna o código mais limpo.